### PR TITLE
Add Sorting + FIX (hasProperty to hasAttribute)

### PR DESCRIPTION
**Pull request by sankam-nikolya**
Add Sorting for rows
hasProperty to hasAttribute (hasProperty return false if use dynamic model)

### CODE REVIEW

**Comment by unclead** [comment]
thx for nice PR
But before i merge it update changelog please

File: src/MultipleInput.php [hunk 1]
```diff
@@ -155,7 +161,7 @@ protected function initData()
         }
 
         if ($this->model instanceof Model) {
-            $data = $this->model->hasProperty($this->attribute)
+            $data = $this->model->hasAttribute($this->attribute)
```
Comment (unclead) [inline src/MultipleInput.php]: better to revert to `hasProperty` because simple models don't have attributes, they have only properties

**Comment by sankam-nikolya** [comment]
sorry find some js error

**Comment by sankam-nikolya** [comment]
i wil fix and change change log

**Pull request closed without merging**